Signal/Images.xcassets/file-icon-large.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "file-icon-large@1x.png",
```

Comment:
I don't know? Assuming you checked in on all the resolutions we're using, I don't have a problem with it.